src/main/java/io/cdap/plugin/gcp/bigquery/actions/BigQueryDropTableAction.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.gcp.bigquery.actions;
+
+import com.google.cloud.bigquery.Table;
+import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Name;
+import io.cdap.cdap.api.annotation.Plugin;
+import io.cdap.cdap.etl.api.action.Action;
+import io.cdap.cdap.etl.api.action.ActionContext;
+import io.cdap.plugin.gcp.bigquery.util.BigQueryUtil;
+
+import java.util.Objects;
+
+/**
+ * This class <code>BigQueryDropTableAction</code> is a plugin that would allow users
+ * to delete BigQuery tables.
+ */
+@Plugin(type = Action.PLUGIN_TYPE)
+@Name("BigQueryDropTable")
+@Description("This action drops given BigQuery table.")
+public class BigQueryDropTableAction extends Action {
```

Comment:
add config validation

Reply:
Project ID, dataset and table name are not validated in rest of BQ plugin.

Reply:
that does not mean we should not validate them